changelog.txt

```diff
@@ -0,0 +1,3 @@
+important - Running "firebase deploy" with database functions that have not been updated since February 2017 will result in a "provider change not allowed" error. To work around this, comment out the functions, run "firebase deploy" to delete them, uncomment the functions, then run "firebase deploy" to re-create them.

```

Comment:
I wonder if this would be better removed from the changelog and sent as an email to the trusted testers list, since they're the only ones affected.